Java/README.md

```diff
@@ -1,5 +1,27 @@
-# Tomcat Server for M14
+# M14 (Music14) Backend Java Code
 
-Currently working on our Server, proj-309-m14.cs.iastate.edu
+We currently use [Apache Tomcat](http://tomcat.apache.org/) and [Jersey](https://jersey.java.net/) to run the REST services for user management.
 
-To access, go to: [proj-309-m14.cs.iastate.edu/REST/v1/test/get/0](http://proj-309-m14.cs.iastate.edu/REST/v1/test/get/0)
+At the moment M14 is not yet live. We are looking into hosting solutions.
+
+
+## Development Setup & Practices
+
+We do most of our backend development in [Eclipse](https://www.eclipse.org) which we would recommend.
+
+Maven is used to manage dependencies and build the backend portion of M14.
+
+JUnit tests are appreciated. :)
+
+
+## Java Code Guidelines
+
+We follow the [Google Java Style Guide](http://google-styleguide.googlecode.com/svn/trunk/javaguide.html)
+
+- Brackets follow the Kernighan and Ritchie style with brackets on the same line.
+- `@override` always used.
+- Constant names use `CONSTANT_CASE`.
+- Class names use `UpperCamelCase`.
+- Other variable or method names use `lowerCamelCase`.
+- Javadoc-style comments.
+- Don't be stupid. Others sould be able to read your code.
```

Comment:
Lol out of all the lines, the don't be stupid line has a typo.
![LOL](http://gifrific.com/wp-content/uploads/2014/12/Joe-Biden-Laughing-Shaking-his-Head.gif)